user_specs_mod.f90

```diff
@@ -97,9 +97,9 @@ module user_specs_mod
    character(str_len) :: grid_lon       = 'GLlon_512.txt'           ! Grid file for longitude
    
    ! Model parameters==================================================================================================!
-   integer, parameter :: norder = 512           ! Max spherical harmonic degree/order
+   integer :: norder = 512                      ! Max spherical harmonic degree/order
```

Comment:
Do you want to add to the comments for these two that this is optionally namelist-configurable?

Reply:
I'm hesitant to do that for just this option because there are many other options in the same module that are configurable in the namelist. What I can do is to list all the configurable options at the top of the module?